docker-auth.yaml

```diff
@@ -0,0 +1,5 @@
+username: assignmentexec
+password: AssignmentExec@2020
+repository: trial2
+version: 2.5
+dockerfilePath: Dockerfile
```

Comment:
this has nothing to do with authentication. As mentioned, use environment variables to obtain this information.

Reply:
You mean only the dockerfile pathname, or all the fields of docker-auth. yaml ?

Reply:
Just dockerFilePath.